requirements/requirements.txt

```
@@ -24,9 +24,9 @@ click==8.2.1
     # via uvicorn
 colorama==0.4.6
     # via sphinx-autobuild
-distlib==0.3.9
+distlib==0.4.0
     # via virtualenv
-docutils==0.21.2
+docutils==0.22
```

`myst-parser` doesn't currently work with v 0.22, so for now keep:
```suggestion
docutils>=0.19,<0.22
```